examples/nodejs-nestjs/src/config/arcjet.ts

```diff
@@ -0,0 +1,7 @@
+import { detectBot } from '@arcjet/node';
+
+export const ARCJET_DEFAULT_RULES = [
```

Comment:
Also worth a comment explaining this and how you might adapt it to your own use.